lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -771,5 +810,36 @@ export abstract class ClientApplication {
         return validLogoutRequest;
     }
 
+    /**
+     * Broadcasts events by calling callback with broadcast message
+     * @param eventType 
+     * @param interactionType 
+     * @param payload 
+     * @param error 
+     */
+    broadcastEvent(eventType: BroadcastEvent, interactionType?: InteractionType, payload?: BroadcastPayload, error?: BroadcastError) {
+        const message: BroadcastMessage = {
```

Comment:
We have a few different terms we are using, "broadcast", "event", "message". I think it would be good to not use "broadcast" (as it is Angular nomenclature) and pick one of the other two. Let's chat about it this afternoon.

Reply:
Broker is going to use "message" pretty heavily as well, so it may be good to make sure we can differentiate between event broadcast messages and window frame messages easily.

Reply:
Renamed to EventMessage, EventType, emitEvent after discussion.